android/src/main/java/com/instabug/reactlibrary/RNInstabugReactnativeModule.java

```diff
@@ -134,7 +134,7 @@ public void run() {
      * @param invocationEventValues The events that invoke the SDK's UI.
      */
     @ReactMethod
-    public void init(final String token, final ReadableArray invocationEventValues, final String logLevel) {
+    public void init(final String token, final ReadableArray invocationEventValues, final String codePushLabel, final String logLevel) {
```

Comment:
I wonder where the word `label` came from as we already use `version` on both iOS and Android, is there a reason why it's not just `codePushVersion`?